build/Publish.targets

```diff
@@ -56,13 +53,15 @@
 
     <ItemGroup>
       <ForPublishing>
-        <RelativeBlobPath>$(BlobStoragePartialRelativePath)/$(FullNugetVersion)/$([System.String]::Copy('%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
+        <RelativeBlobPath>$(Product)/$(FullNugetVersion)/$([System.String]::Copy('%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
       </ForPublishing>
```

Comment:
I think you'll also want to add `ManifestArtifactData` as `ShipInstaller=dotnetcli` metadata here ([syntax](https://github.com/dotnet/core-eng/blob/master/Documentation/Orchestrated-Build/Api/publish.md#adding-artifact-information)), so these are copied to `dotnetcli` during [final blob publish](https://github.com/dotnet/core-eng/blob/master/Documentation/Orchestrated-Build/Api/publish.md#publish-runtime-and-sdk-installers-to-known-endpoints).

Version badges should also have `dotnetcli` I think, and checksums will want `ShipInstaller=donetclichecksums`.

Reply:
Lower-case is fine by me...changed.

Reply:
@dagood 
I was going to ask you about this; thanks for the proactive answer.